src/bitwarden_exporter/settings.py

```
@@ -103,17 +104,38 @@ def get_bitwarden_settings_based_on_args() -> BitwardenExportSettings:
         default="bw",
     )
 
+    with open(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), "resources", "cli-help-debug.txt"),
+        "r",
```

Ruff (0.14.1)

Unnecessary mode argument

Remove mode argument

(UP015)